aws_lambda_gateway/src/extensions.rs

```diff
@@ -0,0 +1,7 @@
+use std::collections::HashMap;
+
+#[derive(Debug, PartialEq)]
+pub struct PathParameters(pub HashMap<String, String>);
```

Comment:
I'd prefer not to make the inner hash map `pub`. This will allow us to evolve this over time if we happen to find a more suitable or performant representation.

To provide access to the parameters, maybe you could add a method like `get(&self, name: &str) -> Option<&str>`?

(the same applies for `QueryParameters`)

Reply:
I've updated extensions.rs to abstract from the hash map itself.